pysys/constants.py

```diff
@@ -86,12 +82,15 @@
 	PLATFORM=sys.platform
 	OSFAMILY='unix'
 	DEVNULL = '/dev/null'
-	ENVSEPERATOR = ':'
 	PATH = '/bin:/usr/bin:/usr/sbin:/usr/local/bin'
 	LD_LIBRARY_PATH = '/usr/lib'
 	DYLD_LIBRARY_PATH = ''
 	SITE_PACKAGES_DIR = os.path.join(sys.prefix, "lib", "python%s" % sys.version[:3], "site-packages")
 
+ENVSEPERATOR = os.pathsep
```

Comment:
So "separator" has been mis-spelled in PySys forever now... would be good to fix this, of course keeping the old name around for compatibility.